pages/contact/_hero.html

```diff
@@ -1,9 +1,8 @@
 <div class="container">
   <div class="row">
     <div class="col-xs-12 text-center">
-      <h1>Ask a Grunt</h1>
-      <p style="margin-bottom:4px" class="lead">Speak to a real human!</p>
-      <p>Use the form below or send an email to <a href="mailto:sales@gruntwork.io">sales@gruntwork.io</a>.</p>
+      <h1>Book a Gruntwork Demo!</h1>
+      <p style="margin-bottom:4px" class="lead">Use this form to book a live demo with our DevOps engineers.</p>
```

Comment:
You know, I think we should actually delete the `<p>` tag here. It looks even more clean without it.

<img width="1309" alt="image" src="https://user-images.githubusercontent.com/4295964/226067017-bede146e-6ef0-4aa8-b1d0-50654a084f5a.png">